### ISSUE TITLE
Generate a list of people by location

### ISSUE THREAD

**Issue by fharper** [fharper/coffeechat#22]
Generate a list of people by location.

**Comment by fharper** [comment]
This should wait until there is a longer list of people

**Comment by fharper** [comment]
I removed the city/country information.

**Comment by agardnerIT** [comment]
I just got here and was about to raise an enhancement for exactly this! Being in Australia, most people will probably be outside of my timezone so a visual filter would be useful.

**Comment by fharper** [comment]
You are right, I put back the information in `people.json` and will work on something visual when I have the time.

**Comment by agardnerIT** [comment]
To maintain privacy, perhaps keeping at the `Country` level would be sufficient?

**Comment by fharper** [comment]
Maybe, I'll think about that :)

**Comment by fharper** [comment]
For the sake of saving space in the README, I will probably display the country with flags. One of those projects should be useful:
- https://github.com/lipis/flag-icons
- https://github.com/hampusborgos/country-flags

**Comment by davidgs** [comment]
Maybe adding a `Tiimezone` property would help people decide if they can manage a coffee chat with someone?

**Comment by fharper** [comment]
@davidgs: good idea, can you move this to a new issue please :)
